build_system/seqan3-config.cmake

```diff
@@ -168,6 +169,22 @@ if (NOT SEQAN3_INCLUDE_DIR)
     find_path (SEQAN3_INCLUDE_DIR "seqan3/version.hpp" HINTS ${SEQAN3_DEPENDENCY_INCLUDE_DIRS})
 endif ()
 
+# ----------------------------------------------------------------------------
+# Find any headers that are vendored below our include directory, for example in a Debian Package
+# ----------------------------------------------------------------------------
+
+if (NOT SEQAN3_CLONE_DIR)
+    if (EXISTS "${SEQAN3_INCLUDE_DIR}/seqan3/submodules")
+        file (GLOB submodules ${SEQAN3_INCLUDE_DIR}/seqan3/submodules/*/include)
+        foreach (submodule ${submodules})
+            if (IS_DIRECTORY ${submodule})
+                seqan3_config_print ("  …adding submodule include:  ${submodule}")
+                set (SEQAN3_DEPENDENCY_INCLUDE_DIRS ${submodule} ${SEQAN3_DEPENDENCY_INCLUDE_DIRS})
+            endif ()
+        endforeach ()
+    endif ()
+endif ()
```

Comment:
Can you shortly explain how the folder structure looks like at debian?

Is it 

```
PATH/seqan3/version.hpp
PATH/seqan3/alignment/*
PATH/seqan3/core/*
...
PATH/seqan3/submodules/
```
So basically the content of `seqan3/include/seqan3/` merged with `seqan3/submodules`?

Reply:
The current install script on Debian is
```
include/seqan3 /usr/include/ 
build_system/seqan3-config.cmake /usr/share/cmake/
# range-v3 can be removed once https://github.com/seqan/seqan3/pull/1267 is merged and released
submodules/range-v3/include/range /usr/include/seqan3/submodules/range-v3/include/
submodules/range-v3/include/meta /usr/include/seqan3/submodules/range-v3/include/
submodules/range-v3/include/concepts /usr/include/seqan3/submodules/range-v3/include/
submodules/range-v3/include/std /usr/include/seqan3/submodules/range-v3/include/
submodules/sdsl-lite/include/sdsl /usr/include/seqan3/submodules/sdsl/include/
```

So it is as you assumed, i.e.
```
/usr/include/seqan3/version.hpp
/usr/include/seqan3/alignment/*
/usr/include/seqan3/submodules/rang-v3/*
```

Reply:
The submodules really should be their own packages!

Reply:
@h-2 Well, now that I have packaged a snapshot of the development tree I'll see if we can use the Debian range-v3 package.

As you know, we are all waiting on `sdsl-light` to make a release :crying_cat_face:

Reply:
@eseiler, @mr-c Can you provide the link to the scripts?

Reply:
@marehr https://salsa.debian.org/med-team/seqan3/tree/master/debian

Reply:
Thank you! I replaced this part with the method in #1429.

Reply:
This can now be removed.